File: src/leftbeehind.java
import java.util.Scanner;

class LeftBeehind {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        while (true) {
            int x = s.nextInt();
            int y = s.nextInt();
            if (x == 0 && y == 0) {
                break;
            }
            if (x + y == 13) {
                System.out.println("Never speak again.");
            } else if (x > y) {
                System.out.println("To the convention.");
            } else if (x == y) {
                System.out.println("Undecided.");
            } else {
                System.out.println("Left beehind.");
            }
        }
    }
}

File: src/leftbeehind.go
package main

import "fmt"

func main() {
	var x, y int
	for true {
		fmt.Scan(&x)
		fmt.Scan(&y)
		if x == 0 && y == 0 {
			break
		}
		if x+y == 13 {
			fmt.Println("Never speak again.")
		} else if x > y {
			fmt.Println("To the convention.")
		} else if x == y {
			fmt.Println("Undecided.")
		} else {
			fmt.Println("Left beehind.")
		}
	}
}
